assets/model_monitoring/components/tests/e2e/test_generation_safety_quality_e2e.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""This file contains e2e tests for the data drift model monitor component."""
+
+import pytest
+from azure.ai.ml import MLClient, Output
+from azure.ai.ml.dsl import pipeline
+from azure.ai.ml.exceptions import JobException
+from tests.e2e.utils.constants import (
+    COMPONENT_NAME_GENERATION_SAFETY_QUALITY_SIGNAL_MONITOR,
+    DATA_ASSET_GROUNDEDNESS_PREPROCESSED_TARGET_DATA
+)
+
+
+def _submit_generation_safety_quality_model_monitor_job(
+    ml_client, get_component, experiment_name, production_data
+):
+    generation_safety_quality_signal_monitor = get_component(
+        COMPONENT_NAME_GENERATION_SAFETY_QUALITY_SIGNAL_MONITOR
+    )
+
+    metrics = [
+        "AcceptableCoherenceScorePerInstance",
+        "AggregatedCoherencePassRate",
+        "AcceptableFluencyScorePerInstance",
+        "AggregatedFluencyPassRate",
+        "AggregatedGroundednessPassRate",
```

Comment:
I don't see you add groundtruth column to test data, how can we calcualte groundness metrics w/o this column?